add Circulars Kafka documentation (#2963)
nasa-gcn/gcn.nasa.gov: app/routes/docs.circulars.subscribing.mdx

# Subscribing to Circulars

> ## Kafka Streaming of Circulars via the topic `gcn.circulars`
> 
> If you have an existing Kafka listener generated from the [Start Streaming GCN Notices interface](quickstart), simply add the Kafka topic `gcn.circulars`.
> 
> Alternatively, visit the [Start Streaming GCN Notices interface](quickstart), select JSON, and the Circulars topic.
> 
> ## Receiving Circulars via Email

Reviewer: ```suggestion
## Stream Circulars over Kafka

If you have an existing Kafka listener generated from the [Start Streaming GCN Notices interface](quickstart), simply add the Kafka topic `gcn.circulars`.

Alternatively, visit the [Start Streaming GCN Notices interface](quickstart), select JSON, and the Circulars topic.

## Receive Circulars by Email
```

Also:

- Switch the order of these two sections.
- Add a short paragraph beneath the top level heading (`# Subscribing to Circulars`). Two headings with no intervening text is bad style.
